framework/wazuh/cluster/master.py

```diff
@@ -290,7 +457,7 @@ def end_receiving_integrity_checksums(self, task_and_file_names: str) -> Tuple[b
         received_filename = self.sync_tasks[task_name].filename
         if received_filename == 'Error':
             logger.info("Stopping synchronization process: worker files weren't correctly received.")
-            return
+            return b"Stopping synchronization process: worker files weren't correctly received."
```

Comment:
Delete this change and create another issue to track this. This PR should only contain docstrings updates.